src/components/dropdowns/dropdowns.html

```diff
@@ -0,0 +1,61 @@
+<section class="docs-block">
+    <aside class="docs-block__info">
+        <h3 class="docs-block__header">Dropdowns</h3>
+    </aside>
+    <div class="docs-block__content">
+        <div class="mint-dropdown">
+            <div class="mint-dropdown__label">All options</div>
```

Comment:
Hm, but this label are not inactive. there's gonna be default label or chosen by user.